introduction.md

```diff
@@ -0,0 +1 @@
+My name is Amren Miller and I am learning to code by attending a bootcamp. Nice to meet you all.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.